rpc/personal_api.go

```diff
@@ -111,11 +116,24 @@ func (e *PersonalEthAPI) LockAccount(address common.Address) bool {
 		return true
 	}
 
+	for i, key := range e.ethAPI.keys {
+		if !bytes.Equal(key.PubKey().Address().Bytes(), address.Bytes()) {
+			continue
+		}
+
+		tmp := make([]emintcrypto.PrivKeySecp256k1, len(e.ethAPI.keys)-1)
+		copy(tmp[:i], e.ethAPI.keys[:i])
+		copy(tmp[i:], e.ethAPI.keys[i+1:])
+		e.ethAPI.keys = tmp
+		return true
+	}
```

Comment:
not sure what's the context of this change in relation to the PR title

Reply:
I needed to update the ethAPI keys to allow for it to know about the new account that we are trying to send from

I think this could be improved by having one set of keys for both APIs, I can open an issue for that